app/app.go

```diff
@@ -1038,19 +1038,51 @@ func (app *App) LastLogs(lines int, filterLog Applog) ([]Applog, error) {
 	return logs, nil
 }
 
+type Filter struct {
+	Name      *regexp.Regexp
+	Platform  string
+	TeamOwner string
+	UserOwner string
+}
+
```

Comment:
@fsouza tried here your suggestion for the query method which was way cleaner but somehow api tests were breaking  because the query was returning 0 elements.. don't know all corners of mgo so perhaps I'm missing something here..